client/src/components/Verejne/index.js

```diff
@@ -54,11 +68,9 @@ export default compose(
   }),
   withState('currentPage', 'setCurrentPage', 1),
   withProps(({entities, currentPage}) => ({
-    entities: chunk(reverse(sortBy(entities, ['size'])), VEREJNE_MAX_PAGE_ITEMS)[currentPage - 1] || [],
+    entities:
+      chunk(reverse(sortBy(entities, ['size'])), VEREJNE_MAX_PAGE_ITEMS)[currentPage - 1] || [],
     pageCount: entities ? Math.ceil(entities.length / VEREJNE_MAX_PAGE_ITEMS) : 0,
   })),
-  branch(
-    (props) => !props.entities.length,
-    renderComponent(Loading),
-  )
+  branch((props) => !props.entities.length, renderComponent(Loading))
```

Comment:
Is this formatting change caused by autoformat? Imo it's less readable

Reply:
it's due to autoformat (by the way, set it up so it does not conflict with the script)

I think it's personal preference (I prefer this) - but either way, I gladly submit to our great robot overlords any autoformat that does it for me and spares me from thinking (if you wish to change the way it's setup go ahead, but I'd say it's not worth the time)

Reply:
Yeah, I am 100% FOR automation. Is there some plugin to VS code for that?

Reply:
yup, plugin is called Prettier, and you want "prettier.eslintIntegration": true in settings